### ISSUE TITLE
Tekken Tag Tournament HD (NPEA00326): Corrupt polygons at certain points

### ISSUE THREAD

**Issue by Silanda** [RPCS3/rpcs3#15128]
### Quick summary

Random flat polygons appear at certain points on certain stages and, handily for testing, are front and centre during the first demo after the intro. 

AFAIK they only appear when using the Vulkan backend. However, OpenGL has its own issues which make using that not desirable (see the clouds during the same demo) 
![NPEA00326_screenshot_2024_01_31_14_12_45](https://github.com/RPCS3/rpcs3/assets/4227434/136868b5-fc0c-41b0-8165-2593d547a738)

### Details

As above. Large, flat, polygons appear on screen. During the first demo they appear dead centre of the screen, but they also appear on certain stages at certain angles (e.g. on Eddy Gordo's stage, IIRC). 

Tested using a variety of settings to no avail. The problem is present at default settings.

### Attach a log file

[RPCS3.log](https://github.com/RPCS3/rpcs3/files/14112904/RPCS3.log)

### Attach capture files for visual issues

[NPEA00326_20240131141527_capture.rrc.gz](https://github.com/RPCS3/rpcs3/files/14112909/NPEA00326_20240131141527_capture.rrc.gz)

### System configuration

Windows 11
Ryzen 7 3700x
Geforce RTX 4070Ti
Nvidia Driver 551.23

**Comment by i2puni** [comment]
It's a graphics issue that has been around for a long time and hasn't been fixed.
I hope it gets fixed

**Comment by ChwaraeTeg** [comment]
Can reproduce this on the NA version of the game too [NPUB30590]. Steps to reproduce are boot game, press Start to skip the 'Namco presents' intro movie, then leave it for 5 seconds at the main menu for the first demo to start playing.

Graphic settings (WCB+RCB+WDB+RDB+SRM) have no effect. Accurate XFloat, plus the accuracy options in the Advanced tab have no effect. Issue is present both with LLVM recompiler and the static intepreter.

**Comment by chane2k1** [comment]
This may be an Nvidia driver specific issue. I have the same problem on my computer with an RTX 4070 Ti, but on my other computer with an RX 7900 XT these issues don't appear at all.

**Comment by digant73** [comment]
is that still an issue?

**Comment by chane2k1** [comment]
Yes. The issue still presents itself on Windows 11 24h2, RTX 4070 Ti Super now, using the current Nvidia driver 581.57 and RPCS3 18261. I booted with default RPCS3 configuration.

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/0fc834d8-39a0-4009-b5b9-cf0523ea3d4f" />

**Comment by digant73** [comment]
please an update on this issue. Some improvements on vertex explosion have been implemented

**Comment by chane2k1** [comment]
Good morning. Yes, it's still present. My computer hardware changed a little since last, configuration is now Windows 11 25H2, RTX 5070 Ti now, using Nvidia driver 596.21 and RPCS3 0.040-19376 that it auto-updated to just before taking the following picture.

<img width="3840" height="2160" alt="Image" src="https://github.com/user-attachments/assets/10fbc615-6654-4702-9a58-a9920f233fd6" />

I made sure to delete all caches for the game and boot with default settings. It has the same issue on 1st boot and on subsequent boots after shaders had been cached for the scene.
